### ISSUE TITLE
Add support for noticing when a file is just blank

### ISSUE THREAD

**Issue by TheTechRobo** [PyCQA/pylint#3814]
The problem is that when there is a blank `__init__.py`, Pylint seems to take that as a "trailing newline".

### Solution
maybe check if the file is 0 bytes _before_ checking for trailing newlines?

### Additional context
I am using CodeFactor. No .pylintrc is at the root of my repo.

**Comment by dbaty** [comment]
I cannot reproduce this issue. Pylint does not report any warning on an empty file:

```
$ touch __init__.py
$ du -h __init__.py 
0	__init__.py
$ pylint __init__.py
$ echo $?
0
```

I believe that your file does have at leat one empty line, which is hence rightfully seen by pylint as a trailing newline.

**Comment by hippo91** [comment]
@TheTechRobo thanks for the report.
Once again @dbaty thanks for your investigation!
I'll will wait confirmation form @TheTechRobo before closing the issue.

**Comment by TheTechRobo** [comment]
@hippo91 Thanks! Turns out that for some reason GitHub decided to put **8 bytes** in my blank file, **and not show me**. Sorry for that!
